tests/openvino/native/test_bias_correction.py

```diff
@@ -64,7 +64,7 @@ def map_references(ref_biases: Dict) -> Dict[str, List]:
         return mapping
 
     @staticmethod
-    def remove_fq_from_inputs(model: ov.Model) -> ov.Model:
+    def remove_fq_from_inputs(model: ov.runtime.Model) -> ov.runtime.Model:
```

Comment:
Why the type hints were fixed and not `ov.Model` instantiation from the tests and the whole NNCF as well?

Reply:
This issue is related to  https://github.com/openvinotoolkit/nncf/pull/2372/files. After this merge we have errors on validation: 
 tests/openvino/native/test_fast_bias_correction.py:54: in TestOVFBCAlgorithm
[2024-02-13T09:04:33.720Z]     def check_bias(model: ov.Model, ref_bias: list):
[2024-02-13T09:04:33.720Z] E   AttributeError: module 'openvino' has no attribute 'Model'
link to example build: https://cje-ir-prod01.devtools.intel.com/nex-swodt-aiworkflowsdevel/blue/organizations/jenkins/NNCF%2FNNCF_PostMerge_CI_not_aborted/detail/NNCF_PostMerge_CI_not_aborted/56/pipeline/83

This PR solves the problems but to be honest, I don't know this repository very well, so I'm asking for help with this fix